autocodesign/localcodesignasset/profile.go

```diff
@@ -29,7 +29,7 @@ func NewProfile(info profileutil.ProvisioningProfileInfoModel, content []byte) a
 		id:             "", // only in case of Developer Portal Profiles
 		bundleID:       info.BundleID,
 		certificateIDs: nil, // only in case of Developer Portal Profiles
-		deviceIDs:      nil, // only in case of Developer Portal Profiles
+		deviceUDIDs:    nil, // (recheck this) only in case of Developer Portal Profiles
```

Comment:
It might be better to add a TODO for this, which better highlights that we need to come back to this code later.